src/FundMe.sol

```diff
@@ -3,7 +3,6 @@
 pragma solidity ^0.8.19;
 // 2. Imports
 
-import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
```

Comment:
This should not be needed anymore, as `AggregatorV3Interface` gets imported in the library `PriceConverter.sol`, correct?

Reply:
@thomasender you'd still need to import it!

```
import {PriceConverter, AggregatorV3Interface} from "./PriceConverter.sol";
```

But, I'd like to keep it imported from:

```
import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol";
```

To keep it as similar as possible to the video. 

Did you try walking through the project to see if this breaks anything?

Reply:
@thomasender in the course this repo is used [smartcontractkit/chainlink-brownie-contracts](https://github.com/smartcontractkit/chainlink-brownie-contracts/blob/main/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol), 
you can see it here [(0:07:46) | Lesson 7: Installing @chainlink/contracts](https://youtu.be/sas02qSFZ74?si=aJrHslUNwtmFIUGJ&t=466) 

command to install @chainlink/contracts dependencies:
```
forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit 
```
this is why this import works
```
import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
```

Reply:
Ok, thanks for clarifying. Will close this PR and continue with the course.